bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -394,7 +402,11 @@ spec:
         hostPath:
           path: /var/lib/ovn/data
       - name: run-openvswitch
-        emptyDir: {}
+        hostPath:
+          path: /var/run/openvswitch
```

Comment:
This makes it so that the container's `/run/openvswitch` would be preserved between runs of different containers, which I don't think we want...

Reply:
So, if I am not mistaken (and I think I would like to "call a friend" on this one: @dcbw ). Those locations contain the OVN DB. Which we do not want to preserve between runs because:

1) I am pretty sure we can not upgrade and will crash when going from non-HA -> HA and vice versa (if we would ever, not sure we would though as non-HA is only in version 4.1/4.2 - tech-preview?)
2) Our source of truth is k8s and we sync directly with it on startup anyways, and thus re-write the DB. 

So, yes, I don't _think_ we want this. 

Why did you add this though @pecameron ? Is it required for 2.12?

Reply:
The unix socket needs to be everywhere it is referenced. This fixed the problem.

Reply:
This actually passes tests. I will back out the change.